dockstore-common/src/test/resources/fixtures/dockstore12.yml

```diff
@@ -12,6 +13,7 @@ workflows:
          - gwas*
   -  name: foobar2
      subclass: cwl
+     publish: no
```

Comment:
Interesting that this works. You can enter true, false, yes, no, including all casing variations (True, NO, etc.)?

Reply:
Yes!
Regex: `y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF`
https://yaml.org/type/bool.html

Reply:
Oh, it's actually a YAML boolean type. I thought it was just a string in YAML terms.